src/consts/documentation.ts

```diff
@@ -19,3 +19,5 @@ export const LEARN_MORE_ABOUT_COMPONENTS = 'https://konflux-ci.dev/docs/building
 export const SECURITY_ENTERPRISE_CONTRACT_POLICY_URL =
   'https://conforma.dev/docs/policy/release_policy.html';
 export const SECURITY_ENTERPRISE_CONTRACT_POLICY_AVAILABLE_RULE_COLLECTIONS_URL = `${SECURITY_ENTERPRISE_CONTRACT_POLICY_URL}#_available_rule_collections`;
+export const LEARN_MORE_ABOUT_LOGIN_IMAGE_REPO =
```

Comment:
is this page not created yet? 👀 

![Screenshot 2025-12-18 at 10 38 34](https://github.com/user-attachments/assets/fa1ed7ae-3525-4729-a46e-02193a9eb32f)